0_download_data/README.md

```diff
@@ -8,19 +13,85 @@ Channels: DAPI (DNA/Nuclei), GFP (Endoplasmic Reticulum), RFP (Cytoplasm)
 
 Cells: Schwann cells
 
-Samples: Two genotypes of the NF1 gene, WT +/+ and Het +/-
+Samples: Two genotypes of the NF1 gene, WT +/+ and Null -/-
 
 Microscope: GE Deltavision Elite
 
 Magnification: 20X
 
 Bit-size: 16-bit
 
-Number of images: 96 images
+Number of images: 
+
+- 96 images (Pilot dataset)
+
+- 384 images (New dataset)
+
+Total memory: 
+
+- 212.9 MB (Pilot dataset)
+
+- 204.2 MB (New dataset - post conversion and cropping)
+```
+
+## File Name Structure
+
+![NF1 Pilot Data Metadata](example_images/NF1_Pilot_Data_Metadata.png "NF1 Pilot Data Metadata")
```

Comment:
Would it make sense to clarify this file name structure as that of the pilot data and converted new data? I recognize this is found within the meta content of the image within markdown, but with a rendered image this may be more difficult to surmise. This comment is based on how I understood the section below `Issue with new dataset` - `2. File naming (metadata)` (which I may have misunderstood).

Reply:
hmmm okay maybe I can change the name to "Standard Metadata Name Structure". 

@d33bs Would that make it clearer in your opinion?

Reply:
@jenna-tomkinson - thanks, this does sound more clear to me 👍